src/locale/languages/de.js

```diff
@@ -206,6 +206,7 @@ export const de = {
       starred: 'Starred',
       watching: 'Watching',
       watchers: 'Watchers',
+      topicsTitle: 'THEMEN',
```

Comment:
We tend to leave the new strings in english until a native speaker can give them some love.

Reply:
Yep! I tried to translate some of them on Google Translate, but it might not be so accurate. I'll leave them in English so. 👍